Azure/communication-ui-library
# Allow a range of @azure/communication-calling and @azure/communication-chat versions as peerDependency

For stable releases of `@azure/communication-react` we would like clients to be able to use any stable version of `@azure/communication-calling` and `@azure/communication-chat` beyond a minimum supported version.

Tested by

* Run `node common/scripts/force-build-flavor.mjs stable` to make sure that release branch creation continues to work.
* Create an NPM pack and install in a test app to see that the latest stable version of calling SDK is allowed.
